src/lib/core/TLVReader.cpp

```diff
@@ -15,24 +15,32 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+#include <lib/core/TLVReader.h>
 
-/**
- *    @file
- *      This file implements a parser for the CHIP TLV (Tag-Length-Value) encoding format.
- *
- */
+#include <stdint.h>
+#include <string.h>
```

Comment:
Is there a reason why sometimes included the standard lib headers is above SDK headers and sometimes it is not? I would like to know why it doesn't appear to be consistent

Reply:
No real reason - I just copied and pasted what was already there without trying to re-order. I belive splitting them from the rest with newlines makes sense. However where they should go I am unclear about.

If forced to make a choice I would put them last to limit chances of accidental dependency on include order, however not sure if I should try to update the PR for that as the updates were mostly automatic with me only adding some whitespace.

Reply:
When I see such inconsistent I'm trying to follow google C++ styleguide: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
+1

I will try to update includes as I go by if changing individual files. For large changes for now though, I would prefer to limit change amount.

Reply:
I guess if we want to standardize this we should do so through automation. I am a little surprised some formatter/linter wasn't already enforcing this.

What you have done is much better than what was there before and doesn't create an issue. Feel free to mark this as resolved